rootfs/ingest/bin/replctl-register

```diff
@@ -94,7 +94,7 @@ if __name__ == "__main__":
     url = urllib.parse.urljoin(args.url, path)
     json = {"database": metadata.database,
             "CAINFO": "/etc/pki/tls/certs/ca-bundle.crt",
-            "SSL_VERIFYPEER": 0
+            "SSL_VERIFYPEER": 1
```

Comment:
That won't be need since the default behavior of the Ingest service is to verify peers. Though, you may still keep this option here for the sake of clarity.

Reply:
:+1: